test/cat.jl

```diff
@@ -137,15 +119,14 @@ module TestCat
     @test isequal(vcat(dtd, dtc0, dtc), dc)
     @test eltypes(vcat(dtd, dtc0)) == eltypes(dc)
 
-    # Missing columns
```

Comment:
I'll check on this. I think I removed this for the same reason as the other one. Either it's a duplicate test or fails due to stricter rules. I think it was an array promotion issue?